vendor/engines/secure_rooms/app/services/secure_rooms/access_handlers/occupancy_handler.rb

```diff
@@ -27,7 +27,7 @@ def process
           )
         else
           # TODO: Add error cases
-          raise NotImplementedError
+          Rails.logger.warn "Encountered Double-Scan Situation"
```

Comment:
Do we want to add any additional information in the log like which controller/user it was? This may be going away soon enough that we don't care.